packages/plugins/content/slate/src/migrations/v002.js

```diff
@@ -0,0 +1,31 @@
+import { Migration } from 'ory-editor-core/lib/service/plugin/classes';
+import rename from 'deep-rename-keys';
+const migration = new Migration({
+    version: '0.0.2',
```

Comment:
I think it would be better to define the version ranges here:

```
new Migration({
  toVersion: '0.0.2',
  fromVersionRange: '~0.0.1' // just an example,
  migrate: state => state
})
```

Reply:
This is actually not how it works right now, it's little bit complicated like always with migrations. I don't even believe there's right way to do this.
Basically right now, you as an author should provide all migrations necessary to migrate from older to newer. 
These are sorted by version. That means that if I define migrations for 
0.0.1 
0.0.3
0.0.6
My plugin is version 0.0.6 for example.
My content is version 0.0.2
Then it would apply migration 0.0.3 and 0.0.6
If content was 0.0.0
It would use all 3 migrations.
If I explained this properly, it should make sense that the "fromVersionRange" is not necessary. It's implicitly based on all migrations provided.